source/constructs/api/discovery_job/crud.py

```diff
@@ -167,15 +167,32 @@ def init_run(job_id: int) -> int:
     if job.all_jdbc == 1:
         __add_job_databases(run, DatabaseType.JDBC, base_time_dict)
     for job_database in job_databases:
-        run_database = models.DiscoveryJobRunDatabase(run_id=run.id,
-                                                      account_id=job_database.account_id,
-                                                      region=job_database.region,
-                                                      database_type=job_database.database_type,
-                                                      database_name=job_database.database_name,
-                                                      table_name=job_database.table_name,
-                                                      base_time=job_database.base_time,
-                                                      state=RunDatabaseState.READY.value,
-                                                      uuid=uuid.uuid4().hex)
+        print(f"job_database.database_name:{job_database.database_name}")
+        print(f"job_database.table_name:{job_database.table_name}")
+        if is_empty(job_database.database_name) and is_empty(job_database.table_name):
+            catalog_databases = get_catalog_database_level_classification_by_params(job_database.account_id,job_database.region,job_database.database_type).all()
+            for catalog_database in catalog_databases:
+                base_time = base_time_dict.get(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `get` method without a default argument to return the value of a key in a dictionary. We recommended that you use a default argument so that if the value for your key is not found, a default value is returned. If a default value is not provided and the key is not found, then `None` is returned.

[Learn more](https://docs.python.org/3/library/stdtypes.html#dict.get)